ui/nuxeo-page-provider-display-behavior.js

```diff
@@ -463,8 +517,11 @@ export const PageProviderDisplayBehavior = [
     _updateFlags() {
       this.size = Array.isArray(this.items) ? this.items.length : 0;
       const selectedItemsSize = Array.isArray(this.selectedItems) ? this.selectedItems.length : 0;
-      this._isSelectAllIndeterminate = selectedItemsSize < this.size;
-      this._isSelectAllChecked = selectedItemsSize === this.size;
+      this._isSelectAllIndeterminate = !this._isSelectAllActive || selectedItemsSize < this.size;
+      /**
+       * Selecting all documents manually is different than using select all mode
+       */
+      // this._isSelectAllActive = this._isSelectAllActive || (selectedItemsSize === this.size && this.size !== 0);
```

Comment:
Maybe not worth keeping this.

Reply:
Yes, I thought about it too, but ended up keeping it as a reminder that having all documents selected, does mean select all is active. But maybe I can remove it, if it doesn't make sense.